metricbeat-tests/runner_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // stackVersion is the version of the stack to use
 // It can be overriden by OP_STACK_VERSION env var
-var stackVersion = "7.4.0"
+var stackVersion = "7.6.0"
```

Comment:
💯

Reply:
Is that code reading secrets from vault? If so, then it would make it more complex reading that value from Go code. If not, I'm happy to add an HTTP request to get the value

Reply:
No, as far as I know, it's publicly reachable

Reply:
Cool then! Let me add it as a follow-up issue

thanks!